.prettierignore

```diff
@@ -1,4 +1,5 @@
 .coverage
+.vscode
```

Comment:
I'm fine with this, though it does peek my interest, should we add some VS Code settings to the repo?

Reply:
> I'm fine with this, though it does peek my interest

Ditto. For example, I wonder if it'll be helpful to include a launch file for debugging when running jest?

Reply:
We should not, because I don't want anyone changing setting on my computer. And I think many people don't want it either. I have test cases in `.vscode` when trying to debug something, for example.